src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/Field.js

```diff
@@ -57,9 +58,25 @@ export default class Field extends React.PureComponent<Props> {
     }
 
     render() {
-        const {error, value, locale, schema, showAllErrors} = this.props;
+        const {error, value, locale, schema, showAllErrors, name} = this.props;
         const {label, maxOccurs, minOccurs, options, required, type, types} = schema;
-        const FieldType = fieldRegistry.get(type);
+        let FieldType;
+
+        try {
+            FieldType = fieldRegistry.get(type);
+        } catch (e) {
```

Comment:
Didn't we say it would be better to use the error boundary functionality of react?

Reply:
I think it's not possible here